include/align.h

```diff
@@ -34,11 +34,13 @@ typedef struct align { /* alignment & record */
 /* Some altars are considered shrines, add a flag for that
    for the altarmask field of struct rm. */
 #define AM_SHRINE       0x08
+/* High altar on Astral plane or Moloch's sanctum */
+#define AM_SANCTUM      0x10
```

Comment:
I can see the choice of `AM_SANCTUM` misleading people to think it _only_ means the Sanctum altar, since nothing in the game calls the temples on the Astral Plane sanctums. Shouldn't it be `AM_HIGH` or `AM_HIGH_ALTAR` to correspond with the usual terminology?

Reply:
Yeah, I know what you mean... the reason I called it `AM_SANCTUM` is because that's the altar type in the level files (it dates back to the old des format):

https://github.com/NetHack/NetHack/blob/721b1cc9b92a802cdf542a09404c8562e16799d1/dat/astral.lua#L82-L91

Reply:
I wasn't sure about it either, but since they're called 'sanctums' internally I figured this would more consistent with the rest of the level generation stuff...  You can see another reference here:

https://github.com/NetHack/NetHack/blob/721b1cc9b92a802cdf542a09404c8562e16799d1/src/sp_lev.c#L2330-L2334

Reply:
I'd argue this is a good opportunity to adjust sp_lev.c to accept "high" instead of "sanctum" for high altars.

Reply:
I mentioned this on IRC but just for the record: "high" or "highaltar" does seem like a more intuitive name for the type, though that change feels a bit outside the intended scope of this PR.  I'll definitely update it to be consistent with the internal terminology if/when it changes.